src/lib/core/components/filter/TableFilter.tsx

```diff
@@ -93,17 +93,22 @@ export function TableFilter({
           });
         }
       );
+      //DKDK by setting union for label (array number or string), ts error occurs
       const fgValueByLabel = Object.fromEntries(
-        distribution.foreground.barplot.data[0].label.map((label, index) => [
-          label,
-          distribution.foreground.barplot.data[0].value[index] ?? 0,
-        ])
+        distribution.foreground.barplot.data[0].label.map(
+          (label: string | number, index: number) => [
```

Comment:
See the comment on data-api.ts - **if** we roll that back, don't forget to roll this back.

Reply:
@bobular as you said, this is related to above. If we can confirm with Danielle whether the label is either string[] or number[], then I can use single type for label in there (and at data-api.ts).